hal/lib/shared/HAL.cpp

```diff
@@ -18,17 +18,22 @@ struct HAL_JoystickAxesInt {
 
 extern "C" {
 
-int HAL_GetControlWord(HAL_ControlWord* data) {
-  return FRC_NetworkCommunication_getControlWord((ControlWord_t*)data);
+HAL_ControlWord HAL_GetControlWord() {
```

Comment:
I wonder how efficient this, is as it creates an extra copy of the struct.  HAL_ControlWord is theoretically a uint32_t but compilers can do weird things with bitfield structures.